lib/v4/errorMap/tooSmall.ts

```diff
@@ -0,0 +1,65 @@
+import type * as zod from 'zod/v4/core';
+import type { AbstractSyntaxTree } from './types.ts';
+
+export function parseTooSmallIssue(
+  issue: zod.$ZodIssueTooSmall
+): AbstractSyntaxTree {
+  switch (issue.origin) {
+    case 'number':
+    case 'int':
+    case 'bigint': {
+      return {
+        type: issue.code,
+        path: issue.path,
+        message: `number must be greater ${
+          issue.inclusive ? 'or equal to' : 'than'
+        } ${issue.minimum.toLocaleString()}`,
+      };
+    }
+    case 'date': {
+      return {
+        type: issue.code,
+        path: issue.path,
+        message: `date must be ${
+          issue.inclusive ? 'later or equal to' : 'later to'
+        } "${new Date(issue.minimum as number).toLocaleString()}"`,
```

Comment:
question: assuming someone is running zod & zod-validation-error on their browser, the `toLocaleString` will print nicely formatted message in their current timezone. But if they are used in server context, the timezone format might differ from the end-user's one. Is this something we want to address or use the `toString` one instead?

Reply:
An option would be nice. Keep in mind that if our tests have snapshots against that file, we might end up with flaky tests depending on the timezone their tests are running on. I had issues like this in the past in another repo, hence my asking.

Reply:
Implemented with [bfadc5e](https://github.com/causaly/zod-validation-error/pull/501/commits/bfadc5ebaefaeeff1d13fd5f1310d111b3374257)